include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -258,6 +298,14 @@ class ADIOS2IOHandlerImpl
 
 namespace detail
 {
+    /*
+     * The following strings are used during parsing of the JSON configuration
+     * string for the ADIOS2 backend.
+     */
+    constexpr char const * const str_engine = "engine";
```

Comment:
global vars: :scream: 

can we place them in a struct or so? e.g. `struct ADIOS2defaults { ... };`?

Reply:
They are `constexpr`, so not really variables ;)
The alternative would be using `#define` or using literals directly in the code, I think this is the cleanest solution. Putting them in a struct is a good idea, will do that.